### ISSUE TITLE
Wrong selection in Mobile DataList with facet header

### ISSUE THREAD

**Issue by cagataycivici** [primefaces/primefaces#81]
when i tap on the row of my dataList (in mobile environment), the wrong Object value is passed to the ajax listener (it's the next value in the dataList).
It happens only if i put a <f:facet name=header> in the list.

Using PF 5.0.1

``` xml
 <p:dataList id="list" var="seg" value="#{segnBean.segnalazioni}"
                emptyMessage="#{l.emptyTable}" pt:data-inset="true">
        <f:facet name="header">
            #{l.elencoSegnalazioni}
        </f:facet>

        <p:ajax event="tap" listener="#{segnBean.selectSegnalazione}" oncomplete="PrimeFaces.Mobile.navigate('#second', {reverse: true|false,transition: 'flip'});"  />

        <h:outputLink value="#">
            <h2>#{seg.segId}
        </h:outputLink>

    </p:dataList>
----------------------------
  public void selectSegnalazione(SelectEvent event) {       
        Segnalazioni seg = (Segnalazioni) event.getObject();       
        System.out.println(seg.getSegId());       
        ...
    }
```

**Comment by sudheerj** [comment]
I tried to replicate on latest showcase(based on 5.3) but due to some reason the listener itself not invoked.

**Comment by melloware** [comment]
PrimeFaces has [announced ](https://www.primefaces.org/primefaces-6-2-roadmap/)it is deprecating Mobile in 6.2 and removing in 6.3. Ticket can be closed as WontFix.

**Comment by tandraschko** [comment]
PrimeFaces Mobile will be deprecated  in 6.2 and removed in 6.3 ( see https://www.primefaces.org/primefaces-6-2-roadmap/)
Therefor i will close this issue as won't fix. If you will provide a PR to fix this issue until 6.2 RC1, i can merge it.
